WordPress/wordpress-develop
# Sync wordpress/block-library package

To include https://github.com/WordPress/gutenberg/pull/44876. From the PR desc:

In dynamic blocks, use a `wp_` prefixed `wp_get_typography_font_size_value` rather than its `gutenberg_` counterpart, `gutenberg_get_typography_font_size_value`.

See https://github.com/WordPress/wordpress-develop/pull/3437#issuecomment-1274846664 for the rationale.

## Testing Instructions
Insert one of the affected blocks (Navigation Link, Navigation Submenu, Page List, Search) and set its font size.
Save and view on the frontend. Confirm that it works, and no fatals are thrown.

Trac ticket: https://core.trac.wordpress.org/ticket/56467

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**
